bokeh/charts/builders/histogram_builder.py

```diff
@@ -111,19 +113,24 @@ class HistogramBuilder(BarBuilder):
 
     """
 
-    bins = Int(default=None, help="""
-    Number of bins to use for the histogram. (default: None,
-    use Freedman-Diaconis rule)
+    bins = Either(List(Float), Int, default=None, help="""
+    If bins is an int, it defines the number of equal-width bins in the
+    given range (10, by default). If bins is a sequence, it defines the
```

Comment:
it says 10 is default, but the default is None, selecting an optimal number of bins.